### PR TITLE
[feat] Added Mapping and Settings in Index Creation request

### PR DESCRIPTION
**Pull request by lubbo**
Added additional information on IndicesCreation request:

- Mappings
- Settings

### CODE REVIEW

**Comment by prafsoni** [comment]
@lubbo this is already fixed in alpha8. Let me know if there is something this pull request fixes it or adds new feature to it.

**Comment by lubbo** [comment]
From where do you make the releases? 
If I remember well you told me that dev branch is the latest where new features should be put... isn’t it?

**Comment by prafsoni** [comment]
Releases are always cut from master. Currently dev has https://github.com/pksprojects/ElasticSwift/pull/23#issuecomment-509091270 changes and the merge with master is unstable so I have not pushed it. After I get it stable I will get away with it since it will One more. branch to keeps in sync when we get to stable releases. Following a common pattern to use master for developing and cutting release branches from it. Will include contribution guidelines alongside alpha9

File: Sources/ElasticSwift/Requests/IndicesRequests.swift [hunk 1]
```diff
@@ -143,9 +159,142 @@ public class CreateIndexRequest: Request {
...
+    
+    public init(properties: [String : IndexMappingProperty]?, type: FieldType?, format: String?, fields: [String : IndexMappingMultiField]?) {
+        self.properties = properties
+        self.type = type
+        self.format = format
+        self.fields = fields
+    }
+}
+
+public enum FieldType : String, Codable {
```
Comment (prafsoni) [inline Sources/ElasticSwift/Requests/IndicesRequests.swift:248]: I believe this enum will be useful addition to alpha8 implementation

**Pull request closed without merging**